dev/custom-pages/research.html

```diff
@@ -0,0 +1,176 @@
+
+<!-- Impact -->
+<section class="callout callout-impact sections-menu-target homepage-section" data-section-link-title="Global Impact" id="impact-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+<h2 class="h1 heading-slab-serif heading-add-underline">Local Research, Global Impact</h2>
+      </div>
+    </div>
+
+<div class="row">
+<div class="col-md-7">
+<p class="lead">Strong relationships and a shared vision lead to successful partnerships right here in Central Florida. One such partner, the Florida High Tech Corridor, spans 23 counties and serves to attract and retain technology industries and a talented workforce in our area. UCF has solid relationships throughout the region that are built on mutual respect and common goals, and it’s clear that together we are making an impact.</p>
+<p class="lead">In nearby Research Park, many of the 10,000 employees work with UCF researchers and students on projects in the sciences, engineering, photonics and optics, modeling and simulation and health-related fields. And our Business Incubator, which provides young companies with the tools they need to become stable and high-growth enterprises, has helped create more than 900 jobs and $170 million of investment.</p>
+</div>
+<div class="col-md-4 col-md-offset-1">
+<h2 class="heading-uppercase"">Research Centers and Institutes</h2>
+<ul class="list-navigation">
+<li><a target="_blank" class="external" href="http://www.ampac.ucf.edu/">Advanced Materials Processing and Analysis Center</a></li>
+<li><a target="_blank" class="external" href="https://med.ucf.edu/biomed/">Burnett School of Biomedical Sciences at the UCF College of Medicine</a></li>
+<li><a target="_blank" class="external" href="https://incubator.ucf.edu/">Business Incubation Program</a></li>
+<li><a target="_blank" class="external" href="http://www.creol.ucf.edu/">Center for Research and Education in Optics and Lasers</a></li>
+<li><a target="_blank" class="external" href="http://crcv.ucf.edu/">Center for Research in Computer Vision</a></li>
+<li><a target="_blank" class="external" href="http://www.floridahightech.com/">Florida High Tech Corridor Council</a></li>
+<li><a target="_blank" class="external" href="http://www.creol.ucf.edu/About/FPCE.aspx">Florida Photonics Center of Excellence</a></li>
+<li><a target="_blank" class="external" href="http://business.ucf.edu/">Florida Small Business Development Center</a></li>
+<li><a target="_blank" class="external" href="http://www.fsec.ucf.edu/en/">Florida Solar Energy Center</a></li>
+<li><a target="_blank" class="external" href="http://fsi.ucf.edu/">Florida Space Institute</a></li>
+<li><a target="_blank" class="external" href="http://www.ist.ucf.edu/">Institute for Simulation and Training</a></li>
+</ul>
+</div></div>
+
+
+  </div>
+</section>
```

Comment:
Need to re-open .container, .row, .col-md-12 at the end of the page (so that closing divs in template-media-feature.php have something to close)